Nautilus/Handlers/PrefabHandler.cs

```diff
@@ -143,7 +143,14 @@ public void Add(PrefabInfo info, PrefabFactoryAsync prefabFactory, PrefabPostPro
         _prefabs.Add(info, prefabFactory);
         _classIdPrefabs.Add(info.ClassID, info);
         _fileNamePrefabs.Add(info.PrefabFileName, info);
-        _techTypePrefabs.Add(info.TechType.AsString(), info);
+
+        // If multiple prefabs sharing the same TechType are patched, only the first one will be spawnable by its TechType.
+        var techTypeString = info.TechType.AsString();
+        if (!_techTypePrefabs.ContainsKey(techTypeString))
```

Comment:
I think this would be better as 
```
_techTypePrefabs[info.TechType.AsString()] = info;
``` 
I think there could theoretically be a use case for actually replacing another prefab by using the same techtype. It's up to you if this is something we should encourage or not, though, as it is a fairly "hacky" thing to do.